okapi-core/src/main/java/org/folio/okapi/MainDeploy.java

```diff
@@ -162,7 +160,7 @@ private boolean parseOptions(String[] args, Handler<AsyncResult<Vertx>> fut) {
   }
 
   private void printUsage() {
```

Comment:
Can you add

    @SuppressWarnings("squid:S106")  // Suppress "Standard outputs should not be used directly"